make sdist is broken

### Steps To Reproduce

```
make download-for-sdist
[...]
[r-none] Error: r is a dummy package and 
[r-none] cannot be installed using the Sage distribution.
make[2]: *** [Makefile:3575: r-SAGE_LOCAL-no-deps] Error 1
make[1]: *** [Makefile:3575: /home/release/Sage/local/var/lib/sage/installed/r-none] Error 2
```

### Expected Behavior

Should be able to build a source tarball

### Actual Behavior

Broken

### Additional Information

_No response_

### Environment

10.5.beta6

### Checklist

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.
- [X] I have read the documentation and troubleshoot guide

--- a/build/pkgs/rpy2/dependencies
+++ b/build/pkgs/rpy2/dependencies
@@ -1,4 +1,4 @@
- r cffi tzlocal pytz jinja2 | $(PYTHON_TOOLCHAIN) pycparser $(PYTHON)
+ cffi tzlocal pytz jinja2 | $(PYTHON_TOOLCHAIN) pycparser $(PYTHON)
 
 ----------
 All lines of this file are ignored except the first.

